tests/integration/views/test_dump.py

```diff
@@ -83,6 +83,10 @@ def test_dump_answers_authenticated_with_role_with_answers(self):
         # Enable full dictionary diffs on test failure
         self.maxDiff = None
 
+        # Cannot assert on randomly generated IDs
```

Comment:
There probably should be something that either asserts these keys are present? __Or__ you could use the `side_effect` of mock/patch to generate a predictable pattern of `group_instance_id`s, e.g. something along the lines of (untested)

```python
def uuid_mock():
    i = 0
    while True:
        yield group-{}".format(i)
        i += 1
# sometime later
patch('....', side_effect=uuid_mock()`
```

Reply:
I wasn't convinced I should be using mocks in an integration test, I guess it's still better than not checking those fields.